backend/api/Areas/Keycloak/Controllers/UserController.cs

```diff
@@ -34,10 +36,11 @@ public class UserController : ControllerBase
         /// </summary>
         /// <param name="keycloakService"></param>
         /// <param name="mapper"></param>
-        public UserController(IPimsKeycloakService keycloakService, IMapper mapper)
+        public UserController(IPimsKeycloakService keycloakService, IMapper mapper, IUserService userService)
```

Comment:
@Fosol why are we seeing this warning 4 times?

Reply:
Currently we're seeing this warning during `npm test` as a `warning`.  I don't fully understand it.  It has something to do with the order of state added.